package.json

```diff
@@ -57,12 +57,7 @@
     "@total-typescript/ts-reset": "^0.5.1",
     "@types/jest": "^29.5.12",
     "@types/lodash": "^4.17.7",
-    "@types/node": "^20.14.14",
```

Comment:
I've removed the ESLint dependencies because they should be downloaded by installing the @api3/commons-eslint package. This doesn't seem to break anything so I think it's fine.

Reply:
Except CI failed linting 🙁 Looks to because of [new jest formatting rules](https://github.com/jest-community/eslint-plugin-jest/blob/main/CHANGELOG.md) in `eslint-plugin-jest` that were imported from https://github.com/dangreenisrael/eslint-plugin-jest-formatting